Tasks/NuGetCommand/Common/VstsNuGetPushToolRunner.ts

```diff
@@ -37,15 +37,35 @@ export class VstsNuGetPushToolRunner extends ToolRunner {
 
     public execSync(options?: IExecOptions): IExecSyncResult {
         options = initializeExecutionOptions(options, this.settings);
-
-        return super.execSync(options);
+        let execResult = super.execSync(options);
+        if (execResult.code !== 0) {
+            this._logExecResults(execResult.code, execResult.stderr);
+        }
+        return execResult;
     }
 
     public exec(options?: IExecOptions): Q.Promise<number> {
         options = initializeExecutionOptions(options, this.settings);
 
         return super.exec(options);
     }
+
+    private _logExecResults(exitCode: number, stderr: string){
+        try{
+            console.log("##vso[telemetry.publish area=Packaging;feature=NuGetCommand]%s",
+            JSON.stringify({
+                'SYSTEM_JOBID': tl.getVariable('SYSTEM_JOBID'),
+                'SYSTEM_PLANID': tl.getVariable('SYSTEM_PLANID'),
+                'SYSTEM_COLLECTIONID': tl.getVariable('SYSTEM_COLLECTIONID'),
+                'command': tl.getInput("command"),
+                'arguments': tl.getInput("arguments"),
```

Comment:
Same comments as NuGetToolRunner2

Reply:
Again.  I do not want to include the full set of variables when I'm capturing STDERR from the spawned program.  Specifically, I want to see the STDERR, the command, possibly args, and enough data to JOIN the log back to the telemetry at the beginning of the task.